packages/core/src/pagination/pagination-number.element.scss

```diff
@@ -0,0 +1,29 @@
+@import './../styles/tokens/generated/index';
+
+:host {
+  --height: #{$cds-global-space-9};
```

Comment:
People can set the height but they can't set the width?

Reply:
Removed `cds-pagination-number` as it's no longer necessary.